template.json

```diff
@@ -1049,12 +1049,13 @@
             "driver": "",
             "modificationCategories": {
                 "body": 0,
-                "power_tain": 0,
+                "power_train": 0,
```

Comment:
Changes like this will cause Foundry to 
- create the new model property
- leave the old one as is

In theory we'd have to create a data migration to move power_tain over to power_train, however I doubt the effort to be worth the value, as I suspect not many people using the incomplete modification system. So... I'll point it out within the changelog and leave it to the users.

Reply:
Noted. Thanks for the feedback and apologies for the initial mistake!